src/features/unmarkedWsNoticeView/UnmarkedWsNoticeView.tsx

```diff
@@ -104,7 +104,7 @@ const UnmarkedWsNoticeView = ({
       <Card className={styles.fullWidth}>
         <CardHeader title={t('unmarkedWsNotices.noticeDetails.title').toUpperCase()} />
         <CardBody>
-          <Grid colsCount={3}>
+          <Grid colsCount={4} className={styles.noticeDetailsGrid}>
```

Comment:
This layout is becoming a bit complicated, and does not work well with our intended minimum screen width (1280px).
Should both the invoice button and the sticker information just be in the same column?

<img width="957" alt="image" src="https://user-images.githubusercontent.com/15479131/96266527-690c6900-0fcf-11eb-9e3b-26761af46340.png">

Reply:
The sticker should not appear if no invoice has been sent yet. To make this grid more compact, I removed the chip that had no proper translation and was out of date with the the applications status.

Reply:
Excellent idea 👍